Dockerfile

```diff
@@ -10,6 +10,8 @@ ENV ES_VERSION 5.1.1
 RUN wget -q https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-${ES_VERSION}.tar.gz -O - | tar -xzf -; mv elasticsearch-${ES_VERSION} /elasticsearch && mkdir -p /elasticsearch/config/scripts /elasticsearch/plugins
 RUN /elasticsearch/bin/elasticsearch-plugin install io.fabric8:elasticsearch-cloud-kubernetes:5.1.1
 RUN /elasticsearch/bin/elasticsearch-plugin install com.floragunn:search-guard-ssl:5.1.1-19 && (cd /elasticsearch/plugins/search-guard-ssl/ && wget -q http://repo1.maven.org/maven2/io/netty/netty-tcnative/1.1.33.Fork24/netty-tcnative-1.1.33.Fork24-linux-x86_64-fedora.jar)
+RUN /elasticsearch/bin/elasticsearch-plugin install x-pack && \
+    chmod -R +rx /elasticsearch/config/x-pack
```

Comment:
`/elasticsearch/config/x-pack` needs `+x` otherwise:

```Exception in thread "main" org.elasticsearch.bootstrap.BootstrapException: java.nio.file.AccessDeniedException: /elasticsearch/config/x-pack/users_roles
Likely root cause: java.nio.file.AccessDeniedException: /elasticsearch/config/x-pack/users_roles
	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:84)```

Individual config files don't need +x but still need +r so can split that in two separate lines.